File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/Icd10CodeEstimateAmount.cs

namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class Icd10CodeEstimateAmount
    {
        public int Icd10CodeEstimateLookupId { get; set; }
        public string Icd10Code { get; set; }
        public int ICD10DiagnosticGroupId { get; set; }
        public decimal MedicalMinimumCost { get; set; }
        public decimal MedicalAverageCost { get; set; }
        public decimal MedicalMaximumCost { get; set; }
        public decimal PDExtentMinimum { get; set; }
        public decimal PDExtentAverage { get; set; }
        public decimal PDExtentMaximum { get; set; }
        public decimal DaysOffMinimum { get; set; }
        public decimal DaysOffAverage { get; set; }
        public decimal DaysOffMaximum { get; set; }

    }
}

File: Web/RMA-Web-Shell/projects/shared-models-lib/src/lib/common/icd10-code-estimate-amount.ts
export interface Icd10CodeEstimateAmount {
    icd10CodeEstimateLookupId: number;
    icd10Code: string;
    icd10DiagnosticGroupId: number;
    medicalMinimumCost: number;
    medicalAverageCost: number;
    medicalMaximumCost: number;
    pdExtentMinimum: number;
    pdExtentAverage: number;
    pdExtentMaximum: number;
    daysOffMinimum: number;
    daysOffAverage: number;
    daysOffMaximum: number;
}
